crates/paralegal-policy/src/context.rs

```diff
@@ -462,6 +453,15 @@ impl Context {
             let Some(datasource) = current.typ.as_data_source() else {
 				continue
 			};
+
+            let flow = &self
```

Comment:
*if you want* you can also make this potentially more efficient by getting each flow for each node at the start and push their references into the queue also. That way you have them available here and don't have to look them up every time.

Reply:
done